src/Microsoft.AspNetCore.Hosting/project.json

```diff
@@ -50,7 +52,7 @@
         }
       }
     },
-    "netstandard1.3": {
+    "netstandard1.5": {
```

Comment:
Its caused the whole of aspnetcore's xplat to be at `netstandard1.5`/`4.6.2` which is a full framework pre-release level - which while it isn't a huge problem in itself (as coreclr is 1.5).

However, aspnetcore will be used as a primary example for library authors on what pattern to follow so will likely cause up-targeting across the ecosystem rather than encouraging them to target the lowest netstandard they can.